src/amo/components/SearchResults.js

```diff
@@ -72,18 +47,16 @@ class SearchResults extends React.Component {
       }
     }
 
-    const message = (
-      (<p ref={(ref) => { this.message = ref; }} className={classNames({
-        'visually-hidden': hideMessageText,
-        'SearchResults-message': !hideMessageText,
-      })}>{messageText}</p>)
-    );
-
     return (
       <div ref={(ref) => { this.container = ref; }} className="SearchResults">
-        {resultHeader}
-        {message}
-        {searchResults}
+        <AddonCard addons={results}>
```

Comment:
Is the header/footer coming later? Looks like this should say **Search results for “query term” [extensions | themes]**